js/apps/admin-ui/public/locales/en/common.json

```diff
@@ -215,5 +215,7 @@
   "forbidden_other": "Forbidden, permissions needed:",
   "noRealmRolesToAssign": "There are no realm roles to assign",
   "loadingRealms": "Loading realms…",
-  "customAttribute": "Custom Attribute…"
+  "customAttribute": "Custom Attribute…",
+  "unsavedChangesTitle": "Unsaved changes",
+  "unsavedChangesConfirm": "You have unsaved changes. Do you really want to leave the page?"
```

Comment:
I am not sure these labels belong in the `common` namespace. They only seem to be used in the clients section, so perhaps it would be better to add them to `clients.json`.